examples/ex15p.cpp

```diff
@@ -146,7 +146,7 @@ int main(int argc, char *argv[])
       if (ref_levels > 0) { ref_levels--; }
       mesh->SetCurvature(2);
    }
-   mesh->EnsureNCMesh();
+   mesh->EnsureNCMesh(true);
```

Comment:
Do the same in the serial version?

Reply:
OK

Reply:
Done. Updated also example description in ex15 and ex15p. All runs are now nonconforming.

Reply:
Thanks!